lms/envs/common.py

```diff
@@ -362,7 +362,7 @@
     'ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES': True,
 
     # WIP -- will be removed in Ticket #TNL-4750.
-    'ENABLE_TIME_ZONE_PREFERENCE': False,
+    'ENABLE_TIME_ZONE_PREFERENCE': True,
```

Comment:
Yes.  Add to Sprint.  Cc @cdyer

Sent from my iPhone

> On Jul 18, 2016, at 3:14 PM, Kevin Kim <notifications@github.com> wrote:
> 
> In lms/envs/common.py:
> 
> > @@ -362,7 +362,7 @@
> >      'ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES': True,
> >  
> >      # WIP -- will be removed in Ticket #TNL-4750.
> > -    'ENABLE_TIME_ZONE_PREFERENCE': False,
> > +    'ENABLE_TIME_ZONE_PREFERENCE': True,
> Right? Should we add it to the sprint then?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>